No longer depends on colorbuddy

andersevenrud opened the pull request:
This PR improves startup time significantly by removing the dependence on colorbuddy.

```
035.762  001.940  001.940: sourcing /home/anders/.local/share/nvim/site/pack/packer/start/nordbuddy/colors/nordbuddy.lua
```

vs ~20ms.

Closes #24

andersevenrud committed:
No longer depends on colorbuddy

maaslalani commented:
👏 👏 👏

andersevenrud commented:
@maaslalani 

Just wanted to show off what I did to make it work as expected. I have a back-compat `Color` class here so that the `:light()` calls did not break. Should I write these functions or just hardcode something ?

There's also the question of what to do with the styles, aka `s` as this is just a map with 1:1 strings. Maybe we can just go over the color files and replace that with plain strings -- but then it no longer behaves as `c` and `cs` when you're writing these definitions...so maybe just let things be as is ?

maaslalani commented:
> Should I write these functions or just hardcode something ?

We can just delete the `lighten` calls tbh, or hardcode the lightened colors codes. I don't mind either.

> There's also the question of what to do with the styles, aka s as this is just a map with 1:1 strings. Maybe we can just go over the color files and replace that with plain strings

Thats pretty much what I did in my PR:
https://github.com/maaslalani/nordbuddy/pull/26/files#diff-36c51489421ce700b3ab015e5b787c2eb6497cfebb6a80f66d44741ee9f69756R35-R37

andersevenrud commented:
Updated with:

* Replaced Color class with plain strings
* Replaced light() calls with new palette entries

And as for the color definition files, they're pretty much as they were.

Which way are you leaning on this ? #26 ?

maaslalani commented:
Going to merge this one!

andersevenrud commented:
Cool. Thanks! :)
